### ISSUE TITLE
Opinions on using Farm/Catch at the same time?

### ISSUE THREAD

**Issue by clarkeybear** [finndev/PokeBuddy#1202]
Is this a good idea or recipe to get banned?

**Comment by Erlius** [comment]
Im botting farm and catch at the same time. I just set location at my place and walkspeed 6km/hour. so far so good. level 20 right now. never get softban. botting 3hour than rest for 30 min then continue. Just start botting yesterday.

**Comment by AzTactical** [comment]
that is a great idea to keep it like that.  by anychance do you end up being too far rom pokespots when they spin it?

**Comment by Erlius** [comment]
Nope, I think my bots are perfectly running smooth without any error. I suggest you finding good spot at your place. Then start bot there.  There i attach my Live Location after 3 hours running. I get 12k/hour. Not so great but it safe. You dont have to worry about softban. On Task, I ticked Farm, Catch and Recycle only. I will personally evolve (pidgey and weedle) using Lucky egg for extra Experience. 
![untitled](https://cloud.githubusercontent.com/assets/21071735/17724486/097cfb3a-6475-11e6-88ca-ac1046851cad.jpg)

**Comment by AzTactical** [comment]
Im going to have to try that then , been using pier 39 in Cali , and soft bans happening too often now and im just 1 state away lol

**Comment by Erlius** [comment]
Maybe u walk to fast. Thats why always get softbans. Try 6km/h

**Comment by Grundy4432** [comment]
I can not do both at the same time. Like @Erlius mentioned it may have to do with speed. I can back that up because my waking speed is at 10km/h, trying to farm and catch pokemon cause me to get soft banned after 5 minutes. I ended up doing one task at a time, now I can run it for hours. Hahah
